scripts/webpack.js

```diff
@@ -79,12 +83,13 @@ function targetOutputs() {
   return {
     umd: {
       path: umdPath,
-      filename: `${pkg.name}.js`,
+      filename: pathData => pathData.chunk.name === 'minified' ? `${pkg.name}.js` : `${pkg.name}.[name].js`,
```

Comment:
@erezrokah know if I need to do something similar to this for all the other target ouputs?

Reply:
Let me check how we're using the other targets and follow up